src/Console/Command/MigrateCommand.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace MX\PhinxMigrations\Console\Command;
+
+use Magento\Setup\Console\Command\DbStatusCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\NullOutput;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+/**
+ * @package MX\PhinxMigrations
+ * @author  James Halsall <james.halsall@inviqa.com>
+ */
+class MigrateCommand extends Command
+{
+    protected function configure()
+    {
+        $this
+            ->setName('mx:migrate')
+            ->setDescription('Migrates the database so it is ready for use with the current application code.');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $dbStatusCommand = $this->getApplication()->get('setup:db:status');
+
+        $output->writeln('<info>Checking if setup:upgrade is required...</info>');
+        $exit = $dbStatusCommand->run(new ArrayInput([]), new NullOutput());
```

Comment:
what happens when we add a completely new module does this return the "upgrade required" status?

Reply:
Yeah it does, but that’s OK

Reply:
I just tested it by creating a new module in the src folder and the `setup:db:status` says everything is up to date, while the new module is obviously not registered in the database yet. :thinking:

Reply:
🤔 I will investigate this further

Reply:
From testing you are right, the `setup:db:status` command only tells us if the DB has out-of-date modules and not if it is completely missing a module. Will have to look at something bespoke for this I think.